{"hierarchy":{"paths":[["doc:\/\/OpenAI\/documentation\/OpenAI","doc:\/\/OpenAI\/documentation\/OpenAI\/APIError","doc:\/\/OpenAI\/documentation\/OpenAI\/APIError\/LocalizedError-Implementations"]]},"metadata":{"symbolKind":"property","role":"symbol","roleHeading":"Instance Property","externalID":"s:6OpenAI8APIErrorV16errorDescriptionSSSgvp","fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"text":"errorDescription","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"text":"?","kind":"text"}],"modules":[{"name":"OpenAI"}],"title":"errorDescription","extendedModule":"OpenAI"},"kind":"symbol","identifier":{"interfaceLanguage":"swift","url":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIError\/errorDescription"},"abstract":[{"text":"Inherited from ","type":"text"},{"type":"codeVoice","code":"LocalizedError.errorDescription"},{"type":"text","text":"."}],"schemaVersion":{"minor":3,"patch":0,"major":0},"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"platforms":["macOS"],"tokens":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"text":"errorDescription","kind":"identifier"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"? { "},{"text":"get","kind":"keyword"},{"text":" }","kind":"text"}]}]}],"sections":[],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/openai\/apierror\/errordescription"]}],"references":{"doc://OpenAI/documentation/OpenAI/APIError":{"fragments":[{"text":"struct","kind":"keyword"},{"kind":"text","text":" "},{"text":"APIError","kind":"identifier"}],"abstract":[],"identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIError","title":"APIError","url":"\/documentation\/openai\/apierror","navigatorTitle":[{"text":"APIError","kind":"identifier"}],"kind":"symbol","type":"topic","role":"symbol"},"doc://OpenAI/documentation/OpenAI/APIError/errorDescription":{"type":"topic","role":"symbol","abstract":[],"kind":"symbol","url":"\/documentation\/openai\/apierror\/errordescription","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIError\/errorDescription","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"errorDescription","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}],"title":"errorDescription"},"doc://OpenAI/documentation/OpenAI":{"abstract":[],"identifier":"doc:\/\/OpenAI\/documentation\/OpenAI","url":"\/documentation\/openai","kind":"symbol","type":"topic","title":"OpenAI","role":"collection"},"doc://OpenAI/documentation/OpenAI/APIError/LocalizedError-Implementations":{"identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIError\/LocalizedError-Implementations","role":"collectionGroup","kind":"article","abstract":[],"type":"topic","url":"\/documentation\/openai\/apierror\/localizederror-implementations","title":"LocalizedError Implementations"}}}